blog/docs/articles/getting-started-blog-p0.md

```diff
@@ -16,7 +16,7 @@ concepts of Knative work, we hope that you will also learn something from these
 
 1. [Introduction and Overview (this post!)](/blog/articles/getting-started-blog-p0)
```

Comment:
Starting from 0 might be more intuitive? Right now it might be confusing

Reply:
Markdown doesn't support lists starting at number 0 afaik :(